nextflow_schema.json

```diff
@@ -8,86 +8,293 @@
                     "pattern": "^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$"
                 }
-            },
-            "required": [
-                "input"
-            ],
-            "fa_icon": "fas fa-terminal",
-            "description": "Define where the pipeline should find input data and save output data."
+            }
         },
-        "reference_genome_options": {
-            "title": "Reference genome options",
+        "spectral_library_generation": {
+            "title": "Spectral Library Generation",
             "type": "object",
+            "description": "",
+            "default": "",
             "properties": {
-                "genome": {
+                "generate_spectral_lib": {
+                    "type": "boolean",
+                    "fa_icon": "fab fa-think-peaks",
+                    "description": "Set this flag if the spectral library should be generated using EasyPQP from provided DDA data - identification search results and corresponding raw data."
+                },
+                "library_rt_fdr": {
+                    "type": "number",
+                    "default": 0.01,
+                    "fa_icon": "fab fa-think-peaks",
+                    "description": "PSM fdr threshold to align peptide ids with reference run."
+                },
+                "skip_decoy_generation": {
+                    "type": "string",
+                    "fa_icon": "fab fa-think-peaks",
+                    "description": "Set this flag if using a spectral library that already includes decoy sequences and therefor skip assay and decoy generation."
+                },
+                "min_transitions": {
+                    "type": "integer",
+                    "default": 4,
+                    "description": "Minimum number of transitions for assay"
+                },
+                "max_transitions": {
+                    "type": "integer",
+                    "default": 6,
+                    "description": "Maximum number of transitions for assay"
+                },
+                "prec_charge": {
+                    "type": "string",
+                    "default": "2:3",
+                    "description": "Precursor charge"
+                },
+                "decoy_method": {
+                    "type": "string",
+                    "default": "shuffle",
+                    "description": "Method for generating decoys",
+                    "enum": [
+                        "shuffle",
+                        "pseudo-reverse",
+                        "reverse",
+                        "shift"
+                    ]
+                }
+            }
+        },
+        "openswathworkflow_options": {
```

Comment:
OpenSWATH workflow recently changed the defaults in OpenMS to better match modern instruments etc. maybe worth a look. If you've not already done so https://github.com/OpenMS/OpenMS/pull/4153/files

Reply:
I think these changes are already included in the optimal parameters from george